test/metabase/db/custom_migrations_test.clj

```diff
@@ -1995,3 +1996,21 @@
           :date_joined   :%now})
         (migrate!)
         (is (false? (sample-content-created?)))))))
+
+(deftest decrypt-cache-settings-test
+  (impl/test-migrations "v50.2024-06-12T12:33:07" [migrate!]
+    (encryption-test/with-secret-key "whateverwhatever"
+      (mdb.query/update-or-insert! :model/Setting {:key "enable-query-caching"} (constantly {:value "true"}))
+      (mdb.query/update-or-insert! :model/Setting {:key "query-caching-ttl-ratio"} (constantly {:value "100"}))
+      (mdb.query/update-or-insert! :model/Setting {:key "query-caching-min-ttl"} (constantly {:value "123"})))
```

Comment:
I see this was an existing issue with the tests before, and should not block the PR, but just like migrations, migration tests shouldn't depend on toucan model code because they are subject to change. Dependencies of migrations and their tests should be limited to things unlikely to change because migration bugs have such a high impact. So this should be replaced with:
```
(encryption-test/with-secret-key "whateverwhatever"
      (t2/insert! :setting [{:key "enable-query-caching", :value (encryption/maybe-encrypt "true")}
                            {:key "query-caching-ttl-ratio", :value (encryption/maybe-encrypt "100")}
                            {:key "query-caching-min-ttl", :value (encryption/maybe-encrypt "123")}]))
```

Reply:
oh, indeed! I was too caught up with other stuff, will update the code right now